sound/soc/codecs/max98373-i2c.c

```diff
@@ -446,6 +446,7 @@ static bool max98373_volatile_reg(struct device *dev, unsigned int reg)
 	case MAX98373_R2054_MEAS_ADC_PVDD_CH_READBACK:
 	case MAX98373_R2055_MEAS_ADC_THERM_CH_READBACK:
 	case MAX98373_R20B6_BDE_CUR_STATE_READBACK:
+	case MAX98373_R20FF_GLOBAL_SHDN:
```

Comment:
the commit message is a bit unclear.

Is the description that on a regmap sync we don't restore a value that was modified in hardware? I don't specifically get what the 'Amp ON command has been ignored' would mean.

also the indents should be fixed, this looks like a gitk copy/paste.

Reply:
Thanks. Shall modify the commit message and indentation as well.

Reply:
typo ignnored -> ignored.

Reply:
Fixed typo. Thanks.